samples/subsys/task_wdt/prj.conf

```diff
@@ -0,0 +1,9 @@
+CONFIG_LOG=y
+
+CONFIG_WATCHDOG=y
+CONFIG_WDT_LOG_LEVEL_DBG=y
+CONFIG_WDT_DISABLE_AT_BOOT=y
```

Comment:
Is this option needed? It is not widely supported...

Reply:
According to the discussion in https://github.com/zephyrproject-rtos/zephyr/issues/23282 it's still not finally decided how startup of hardware watchdogs should be handled. In case of STM, the watchdog would be enabled at boot time if this is not set and the example would fail. So I think we have to keep this here until the hardware drivers behave consistently (e.g. keeping manufacturer defaults). Also if the watchdog timeout can be set only once, it should not be set at boot time, but within the task watchdog.